nightguard/StatsViewController.swift

```diff
@@ -31,14 +31,10 @@ class StatsViewController: UIViewController {
     }
     
     override func viewDidAppear(_ animated: Bool) {
+
         super.viewDidAppear(animated)
         
-        // force the display into horizontal orientation
-        UIDevice.current.setValue(UIInterfaceOrientation.portrait.rawValue, forKey: "orientation")
-        UIDevice.current.setValue(UIInterfaceOrientation.landscapeRight.rawValue, forKey: "orientation")
```

Comment:
Not sure why it sets portrait and then landscape. Maybe this is fixing an edge case bug that I'm unable to reproduce?

Reply:
Maybe @dhermanns can give a little help with these lines...